docs/components/customization/user-profile.mdx

```diff
@@ -1,113 +1,31 @@
 ---
-title: <UserProfile /> customization
+title: Add custom pages and links to the <UserProfile /> component
 description: Learn how to add custom pages and include external links within the navigation sidebar of the <UserProfile /> component.
 ---
 
-# `<UserProfile />` customization
+# Add custom pages and links to the `<UserProfile />` component
 
-The [`<UserProfile />`][userprofile-ref] component supports the addition of custom pages and use of external links in the navigation sidebar. 
+The [`<UserProfile />`](/docs/components/user/user-profile) component supports the addition of custom pages and use of external links in the navigation sidebar.
 
-## `<UserProfile.Page />`
+There are two ways to render the `<UserProfile />` component:
+- As a modal that opens when the [`<UserButton />`](/docs/components/user/user-button) is clicked
```

Comment:
```suggestion
- As a modal that opens when the [`<UserButton />`](/docs/components/user/user-button) is clicked.
```
Didn't see anything in the styleguide, but do we add periods to the ends of bullet points? I've been doing it in all my PRs

Reply:
no periods.
https://www.instructionalsolutions.com/blog/bulleted-list-punctuation#:~:text=Use%20a%20period%20after%20a,bullet%20lists%20or%20all%20fragments.

Reply:
@JacobMGEvans I first saw periods at the ends of lists in Clerk's docs 🙃. But glad we have clarity now